zebrad/src/components/inbound.rs

```diff
@@ -98,6 +120,10 @@ impl Service<zn::Request> for Inbound {
                 }
             };
         }
+
+        // Clean up completed download tasks
+        while let Poll::Ready(Some(_)) = self.downloads.unwrap().poll_next(cx) {}
```

Comment:
Problem: `self.downloads` is not a `Stream` so the build fails here with:

```
method not found in `components::inbound::downloads::Downloads<tower::buffer::Buffer<tower::util::BoxService<zebra_network::Request, zebra_network::Response, std::boxed::Box<(dyn std::error::Error + std::marker::Send + std::marker::Sync + 'static)>>, zebra_network::Request>, tower::buffer::Buffer<tower::util::BoxService<std::sync::Arc<zebra_chain::block::Block>, zebra_chain::block::Hash, zebra_consensus::chain::VerifyChainError>, std::sync::Arc<zebra_chain::block::Block>>, tower::buffer::Buffer<tower::util::BoxService<zebra_state::Request, zebra_state::Response, std::boxed::Box<(dyn std::error::Error + std::marker::Send + std::marker::Sync + 'static)>>, zebra_state::Request>>`
```

Reply:
It looks like the type does implement `Stream`, maybe the problem is either

- the type implements `Stream`, but `Stream` is not in scope (via `use futures::Stream`) where `poll_next` is used

or 

- the type implements `Stream`, but only with some bound on its type parameters that isn't being satisfied for some reason